.github/ISSUE_TEMPLATE/storage-migration-checklist-short.md

```diff
@@ -38,13 +38,9 @@ Run all validation checks on Moabs and generate reports.  Note that error detail
 - [ ] note druids for each existing C2M error (from report in /opt/app/pres/preservation_catalog/current/log/reports)
   - [ ] ? Ensure there is a github issue in preservation_catalog for each error/object (mult objects with same error can be in same github issue.)
 
-### CV (checksum validation of all cataloged moabs)
-- [ ] run on storage root ```RAILS_ENV=production bundle exec rake prescat:audit:cv[stor_root_name]```
-  - [ ] requeue failed jobs / ensure no jobs failed via resque GUI https://preservation-catalog-prod-01.stanford.edu/resque/overview
-  - [ ] the run should be finished when all jobs in the `checksum_validation` queue have been worked (queue is back down to 0)
-- [ ] after CV finishes, generate report for objects with error status ```RAILS_ENV=production bundle exec rake prescat:reports:msr_moab_audit_errors[stor_root_name,cv_b4]```
-- [ ] note druids for each existing CV error (from report in /opt/app/pres/preservation_catalog/current/log/reports)
-  - [ ] ? Ensure there is a github issue in preservation_catalog for each error/object (mult objects with same error can be in same github issue.)
+### CV -- we won't run this anew before the migration
+
+We had originally planned to run all objects to be migrated through checksum validation the week before the cutover.  Testing indicates that this won't be possible in the time we have with the computational and storage resourcing we have.  Additionally, checksum validation has been running as expected on production (as of 2020-04-15, our least recently checksum validated object on production was checked at 2020-01-13 11:03:05 UTC).  We have other safeguards and assurances in the migration process and in our normal workflow that should give peace of mind re: integrity of preserved objects (C2M before and after the move, FS block level integrity checking built into the `rsync` invocation that copies content from old to new storage, FS block level integrity checking on the originating NetApp filer).
```

Comment:
this is meant to be a very checklist oriented, without a lot of text blobs.  Maybe move this to the wiki?

Reply:
https://github.com/sul-dlss/preservation_catalog/wiki/Storage-Migration---Additional-Information#cv----why-we-wont-run-this-anew-right-before-the-migration